sql/core/src/test/scala/org/apache/spark/sql/DynamicPartitionPruningSuite.scala

```diff
@@ -1238,7 +1238,10 @@ class DynamicPartitionPruningSuite
 
       val plan = df.queryExecution.executedPlan
       val countSubqueryBroadcasts =
-        plan.collectInPlanAndSubqueries({ case _: SubqueryBroadcastExec => 1 }).sum
+        plan.collectInPlanAndSubqueries({
```

Comment:
Nit: you could also do
```
      val countSubqueryBroadcasts =
        plan.collectInPlanAndSubqueries({ case _: SubqueryBroadcastExec => 1 }).sum
      val countReusedSubqueryBroadcasts =
        plan.collectInPlanAndSubqueries({
          case ReusedSubqueryExec(_: SubqueryBroadcastExec) => 1
        }).sum

      assert(countSubqueryBroadcasts == 1)
      assert(countReusedSubqueryBroadcasts == 1)
```

Reply:
ok, I changed

Reply:
We have `InSubqueryExec` now in DPP, so better keep it.

Reply:
Ok, I've reverted the change.